.gitpod.Dockerfile

```diff
@@ -2,11 +2,11 @@
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
 
-FROM gitpod/workspace-full:latest
```

Comment:
Did you intend to send this upstream to tailscale, or perhaps intend to send it to a different workspace?